jwt/jwt.go

```diff
@@ -0,0 +1,37 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+var secretKey = []byte("farawin")
+
+func CreateToken(id string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"id":  id,
+		"exp": time.Now().Add(time.Hour * 24).Unix(),
+	})
+	tokenString, err := token.SignedString(secretKey)
+	if err != nil {
+		return "", err
+	}
+	return tokenString, nil
+
+}
+
+func ValidateToken(tokenString string) error {
+	token, err := jwt.Parse(tokenString, func(token *jwt.Token) (interface{}, error) {
+		// _, ok := token.Method.(*jwt.SigningMethodECDSA)
+		return secretKey, nil
+	})
+	if err != nil {
+		return err
+	}
+	if !token.Valid {
+		return fmt.Errorf("invalid token")
```

Comment:
you can use errors.New instead of fmt.Errorf
cause the latter formats string and you don't formatting string here!
also you can create a variable for `"invalid token"` like `var ErrInvalidToken = errors.New("Invalid Token")`

Reply:
I need more explanation to comprehend the difference and how to use it